### DISCUSSION TITLE
[IntelliSense] How do I support intellisense inside Python files ?

### DISCUSSION THREAD

**Discussion by bitplorer** [tailwindlabs/tailwindcss#10945] [Ideas]
So I am running tailwindcss directly using python files natively and everything is running fine, my css is compiling and all is well except that I don't get the TailwindCSS intellisense inside my python files. 

here is a part of config file: 
```js
content: {
        files:[
        "../../demosite/*.{html,py}",
        "../../demosite/**/*.{html,py}",
        "../../demosite/**/**/*.{html,py}",
        ]
```
now all I wish now is intellisense for python files, how do I do that ?

**Comment by wongjn** [comment]
You'll want to look at configuring [`tailwindCSS.includeLanguages`](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss#tailwindcss.includelanguages), maybe [`tailwindCSS.classAttributes`](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss#tailwindCSS.classAttributes) and perhaps [`tailwindCSS.experimental.classRegex`](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).

**Reply by bitplorer** [reply]
Sure thanks, how should I add `py` indside [special languages](https://github.com/tailwindlabs/tailwindcss-intellisense/blob/f3967a72adbac5c920a6f1884a5dfa6e3097b0bd/packages/tailwindcss-language-service/src/util/languages.ts#L57)  as given here. I see no support for python files in this file.

**Reply by wongjn** [reply]
You'd use [`tailwindcss.includelanguages`](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss#tailwindcss.includelanguages) to treat python files as a language that the extension already supports.

**Reply by bitplorer** [reply]
so I should add 
```js
{
  "tailwindCSS.includeLanguages": {
    "python": "py"
  }
}
```

**Reply by wongjn** [reply]
No, you'd want the value of `python` to be one of the languages defined in the [supported languages](https://github.com/tailwindlabs/tailwindcss-intellisense/blob/master/packages/tailwindcss-language-service/src/util/languages.ts) that is closest to the syntax of the python files you have. For example:
```json
{
  "tailwindCSS.includeLanguages": {
    "python": "html"
  }
}
```

**Reply by bitplorer** [reply]
Okay thanks, let me try

**Reply by bitplorer** [reply]
Thanks am gettings hovers now but not the completions. :)

**Reply by bitplorer** [reply]
Thanks again, Its all working after the reload. Yay! :smiley:

**Reply by bitplorer** [reply]
Just wanted to ask one thing, do I have to always press `ctrl+space` to get auto completion ?
